### DISCUSSION TITLE
Middle Click Pulls From Shulker

### DISCUSSION THREAD

**Discussion by nickadactyl** [tr7zw/ItemSwapper#89] [Ideas]
Quick idea I just now while playing, if you could pull items from shulkers in your inventory by middle-clicking I think that would help the inventory issue a ton. It's another way to avoid constantly placing shulkers down while you're mid build.

**Comment by Vaskyy** [comment]
Kinda the same thing but automatic would be Handrestocking out of the Box.
Would love this feature.

**Comment by tr7zw** [comment]
Moved to #115
